daq/host.py

```diff
@@ -452,7 +464,8 @@ def _docker_test(self, test_name):
         }
 
         self.test_host = docker_test.DockerTest(self.runner, self.target_port, self.devdir,
-                                                test_name)
+                                                test_name,
+                                                timeout_sec=self._get_test_timeout(test_name))
```

Comment:
don't include function-call inline (just harder to read), extract to local variable and then can combine on same line as test_name -- same number of overall lines but easier to read.